### PR TITLE
Add ability to specify login information for Helm repository

### PR DESCRIPTION
**Pull request by mihuross**

### CODE REVIEW

File: chart-updater.py [hunk 1]
```diff
@@ -55,6 +55,8 @@ def healthz():
 )
 @click.option("--git-ssh-identity", help="Git config SSH identity file (key).")
 @click.option("--helm-repo-url", required=True, help="Helm repo URL.")
+@click.option("--helm-repo-user", help="Helm repo user to login.")
+@click.option("--helm-repo-password", help="Helm repo user to login.", default=lambda: os.environ.get("HELM_REPO_PASSWORD", ""))
```
Comment (pinkavaj) [inline chart-updater.py]: NIT `help="Helm repo user to login."` The description should likely mention password?

**Review by pinkavaj — CHANGES_REQUESTED** [review]

File: chart-updater.py (hunk 1 above)
Comment (mihuross) [reply chart-updater.py]: yeah, fixed.

File: chart-updater.py [hunk 2]
```diff
@@ -55,6 +55,8 @@ def healthz():
 )
 @click.option("--git-ssh-identity", help="Git config SSH identity file (key).")
 @click.option("--helm-repo-url", required=True, help="Helm repo URL.")
+@click.option("--helm-repo-user", help="Helm repo HTTP Auth user.")
+@click.option("--helm-repo-password", help="Helm repo HTTP AUth password.", default=lambda: os.environ.get("HELM_REPO_PASSWORD", ""))
```
Comment (pinkavaj) [inline chart-updater.py]: NIT: `AUth` -> `Auth`, pls.

**Review by pinkavaj — CHANGES_REQUESTED** [review]
Pls can you fix that small typo?

**Pull request merged**